src/test/todotest.test.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @jest-environment jsdom
+ */
+const TodolistClass = require('./todotest.js');
+
```

Comment:
- [ ] Good job creating this `todotest.test.js` as required in the project requirement :+1: But from the project requirement you should test the add item and delete item functions used in your program but you have created a file  `todotest.js` with these functions that are not used in your program and this means you are testing your own functions but not the one that makes the program function. :relaxed: I think is better you require and test the file with these functions i.e `scripts/todolist.js` used in the program rather than testing code that is not used in the program :smile: